src/celeritas/global/detail/ActionSequence.cc

```diff
@@ -11,6 +11,7 @@
 #include <tuple>
 #include <type_traits>
 #include <utility>
+#include <celeritas/global/CoreParams.hh>
```

Comment:
should be quotes not `<>`

Reply:
my editor somehow decided to auto-include using angle brackets...